Day-I-Social-Card/social-card/src/App.css

```diff
@@ -1,4 +1,24 @@
-/* 
-  Write your styles/CSS for App.js here and import into the component that you need 
-  IF you'd like to attempt to use less you'll have to research how to do this with Create React App
-*/
+@import './components/HeaderComponents/Header.css';
+@import './components/CardComponents/Card.css';
```

Comment:
Nice use of the CSS import statements, but they are unnecessary in React since all the CSS will be compiled into index.css in the end.